lib/sorcery/test_helpers/rails/request.rb

```diff
@@ -0,0 +1,20 @@
+module Sorcery
+  module TestHelpers
+    module Rails
+      module Request
+        # Accepts arguments for user to login, the password, route to use and HTTP method
+        # Defaults - @user, 'secret', 'user_sessions_url' and http_method: POST
+        def login_user(user = nil, password = 'secret', route = nil, http_method = :post)
```

Comment:
I wonder if secret being the password default makes the most sense here...I can't think of anything that would make more sense off the top of my head though, so I think it should be fine for now.

Reply:
I agree with the password default value. I added modification from Integration#login, which forces us to use ‘secret’. I don’t think the deafault value isn’t needed because what is really needed is to choose the password value, and I will remove the value if necessary.